core/templates/dev/head/pages/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -106,9 +103,11 @@ oppia.factory('ThreadDataService', [
       $http.get(_THREAD_HANDLER_PREFIX + threadId).then(function(response) {
         var allThreads = _data.feedbackThreads.concat(_data.suggestionThreads);
         for (var i = 0; i < allThreads.length; i++) {
-          if (allThreads[i].thread_id === threadId) {
+          if (allThreads[i].threadId === threadId) {
             allThreads[i].messages = response.data.messages;
-            allThreads[i].suggestion = response.data.suggestion;
+            if (!constants.USE_NEW_SUGGESTION_FRAMEWORK) {
+              allThreads[i].suggestions = response.data.suggestions;
```

Comment:
This seems incorrect right? It used to be just "suggestion"

Reply:
oops yeah. I forgot to test the old framework. Will do that now.